ang/civicase/case/actions/services/go-to-webform-case-action.service.js

```diff
@@ -40,16 +40,17 @@
      * @returns {boolean} - true if action is allowed, false otherwise.
      */
     function isActionAllowed (action, cases, attributes) {
-      return checkIfWebformContainsCaseTypeId(action, cases[0].case_type_id);
+      return checkIfWebformVisible(action, cases[0].case_type_id);
     }
 
     /**
      * @param {object} webform webform action object
      * @param {string} caseTypeID case type id
      * @returns {boolean} if webform contains sent case type id
      */
-    function checkIfWebformContainsCaseTypeId (webform, caseTypeID) {
-      return webform.case_type_ids.indexOf(caseTypeID) !== -1;
+    function checkIfWebformVisible (webform, caseTypeID) {
+      return webform.case_type_ids.length === 0 ||
+        webform.case_type_ids.indexOf(caseTypeID) !== -1;
```

Comment:
can you please update the return description of this function to point out to this scenario.